programs/anchor-movie-review-program/src/lib.rs

```diff
@@ -159,6 +159,17 @@ pub struct MovieAccountState {
     pub description: String,
 }
 
+/*
+    8 bytes for the anchor discriminator 
+    32 bytes for reviewer Pubkey 
+    1 byte for the rating
+    4 bytes for the title String (still need to add String length in the account initialization)
+    4 bytes for the description String (still need to add String length in the account initialization),
+ */
+impl Space for MovieAccountState {
+    const INIT_SPACE: usize = 8 + 32 + 1 + 4 + 4;
```

Comment:
Same question re: `#[derive(InitSpace)]`

Reply:
#[derive(InitSpace)] could definitely be used. However, we would have to give a max length for the String in order for it to work.